common/src/main/java/com/box/l10n/mojito/LocaleMappingHelper.java

```diff
@@ -0,0 +1,48 @@
+package com.box.l10n.mojito;
+
+import com.google.common.base.Splitter;
+import com.google.common.collect.HashBiMap;
+
+import java.util.Map;
+
+public final class LocaleMappingHelper {
```

Comment:
knit: is this refactoring needed in this commit? my guess is that it should just change the webapp. 

Also I see you're moving things into static helper (vs spring component). Overall in the project we've been mainly using components (even for small utils). I don't have a bulletproof definition on when a component should be used but for consistency sake I think we should keep using spring component. I'm happy to have a chat if you have a strong argument for one or the other.

Reply:
We need it as we don't have access to CLI code in the webapp, so it was just easier to have these two methods in the common module.

I've updated the class for it to be a spring component as well.

Reply:
oh yeah, I tried to check if the file was moved into common and couldn't see it ... I need new 👓,  my bad.